ctags/parsers/rest.c

```diff
@@ -66,22 +69,20 @@ static NestingLevel *getNestingLevel(const int kind)
 static void makeRestTag (const vString* const name, const int kind)
 {
 	const NestingLevel *const nl = getNestingLevel(kind);
+	int r = CORK_NIL;
 
 	if (vStringLength (name) > 0)
 	{
+		tagEntryInfo *parent = getEntryOfNestingLevel (nl);
```

Comment:
I've added a patch to remove those for now. If we need the functions in the future after we start syncing parsers, we can grab them from upstream.

Reply:
It's incorrect, because I stated in my commend above, we still have to call `getNestingLevel()` because it's that function that calls `nestingLevelsPop()` when appropriate.  Just removing the call removes this side effect, resulting in never removing nest levels.

This is pretty much exactly what I tried to warn about in the comment above when sating that *"have a tendency to remove [unused variables]"*, suggesting it's not so simple here.  What we can do is call the function without storing its value, and possible rename it to be clearer.

Reply:
Ah, OK, I misunderstood (and sadly didn't read the code enough and fell to the trap). I'll fix that and re-push this patch.

Reply:
Done.